README.md

```diff
@@ -1,10 +1,15 @@
-# export-service-go
+# Export Service
+
+The export service lets users request an archive of their data from various services on the platform, allowing them to explore and use their data with their own tools. Because many console dot applications export data for similar purposes, this service provides the necessary functionality that is common to export as a feature.
```

Comment:
Looks great! This services summary is hard to write and gets verbose, maybe change it to something like:

"The Export Service allows users to upload and request data archives, so they can utilize their own tooling. Because many ConsoleDot applications need to export data, this service provides some common functionality."